deploy_website.sh

```diff
@@ -30,6 +30,26 @@ cd $DIR
   :okhttp-urlconnection:dokka \
   :okhttp:dokka
 
+# Dokka filenames like `-http-url/index.md` don't work well with MkDocs <title> tags.
+# Assign metadata to the file's first Markdown heading.
+# https://www.mkdocs.org/user-guide/writing-your-docs/#meta-data
+title_markdown_file() {
+  TITLE_PATTERN="s/^[#][#]* *\(.*\)/title: \1 - OkHttp/g"
```

Comment:
```suggestion
  TITLE_PATTERN="s/^[#]+ *\(.*\)/title: \1 - OkHttp/g"
```

Why g? all instances rather than first?

Reply:
Just habit.

Reply:
Oh and I got `+` to work but I needed the `-E` flag because sed is from the 1970s